packages/extension/src/components/pages/EditWallet/ShowSecret/ShowSecret.tsx

```diff
@@ -25,15 +25,15 @@ export const ShowSecret: FC<ShowSecretProps> = ({ seed, mnemonic, onBackButton }
   const [isCopied, setIsCopied] = useState(false);
   const { password } = useWallet();
   const setTimeout = useTimeout(2000);
-
+  const passwordRef = useRef<HTMLInputElement | null>(null);
   const secretType: Secret = useMemo(() => (seed ? 'seed' : 'mnemonic'), [seed]);
 
   const handlePasswordChange = useCallback(() => {
     setPasswordError('');
   }, []);
 
   const handleConfirmPassword = useCallback(() => {
-    const passwordValue = (document.getElementById('password') as HTMLInputElement).value;
+    let passwordValue = passwordRef.current?.value;
```

Comment:
```suggestion
    const passwordValue = passwordRef.current?.value;
```
This variable is actually used only one time so actually no need to have a variable declaration.